channels/views/posts_test.py

```diff
@@ -783,6 +783,22 @@ def test_update_article_cover(
         article.cover_image_small.height / article.cover_image_small.width
     ) == "{0:0.1f}".format(THUMBNAIL_DIMENSIONS.y / THUMBNAIL_DIMENSIONS.x)
 
+    # Now update the article again, with the same cover (sent as URL)
+    image_url = "http://localhost/{}".format(article.cover_image.url)
+    resp = user_client.patch(
+        url, {"cover_image": image_url, "article_content": "foo"}, format="json"
+    )
+    assert resp.status_code == status.HTTP_200_OK
+    article = Article.objects.get(post__post_id=resp.json()["id"])
+    assert image_url.endswith(article.cover_image.url)
+
+    # Remove the image
+    resp = user_client.patch(url, {"article_content": "foo"}, format="json")
+    assert resp.status_code == status.HTTP_200_OK
+    article = Article.objects.get(post__post_id=resp.json()["id"])
+    assert article.cover_image.name == ""
+    assert article.cover_image_small.name == ""
+
```

Comment:
do you think we should also add a test making sure you can issue a PATCH request with a URL string for the cover image?

Reply:
That is tested on lines 786-793, but I can put it in a separate test if you prefer

Reply:
gosh I should really read more closely! haha